components/RichTextEditor.js

```diff
@@ -245,6 +245,15 @@ export default class RichTextEditor extends React.Component {
       this.editorRef.current.addEventListener('trix-attachment-add', this.handleUpload);
       this.editorRef.current.addEventListener('trix-file-accept', this.handleFileAccept);
 
+      // We must listen when the user presses the 'Enter' key and when the user clicks the 'Link' button as well
+      const _this = this;
```

Comment:
@Jeanluis019 : I believe this redefining the `_this` constant is unnecessary? 🤔 

```suggestion
```

Reply:
Since we're using an arrow function, `this` should refer to the parent. I just tried to `console.log(_this, this);` and they're indeed the same.